pkg/controller/utils/common/common.go

```diff
@@ -45,3 +46,37 @@ func LookupHost(host string, logger logr.Logger) (string, error) {
 	logger.Info("host resolved", "host", host, "addresses", addresses)
 	return host, nil
 }
+
+const (
+	// OperandImageOpenJDK envvar name for OpenJDK operand image
+	OperandImageOpenJDK = "RELATED_IMAGE_OPENJDK"
+	// OperandImageOpenJ9 envvar name for OpenJ9 operand image
+	OperandImageOpenJ9 = "RELATED_IMAGE_OPENJ9"
```

Comment:
OpenJ9 and OpenJDK looks like can't coexists in one operator deployment. What is the purpose for this implementation? Looks like one env is enough for this option.
The init container and Infinispan container env vars are separate, but OpenJ9 and OpenJDK are not.

Reply:
see above, this is to be friendly with some build processes that need the list of all RELATED_IMAGES in the CSV